packages/create-starknet/templates/next/pages/_app.tsx

```diff
@@ -0,0 +1,6 @@
+import '@/styles/globals.css'
```

Comment:
I believe this file should have the starknet provider configured and wrapping `<Component />`.